packages/terra-infinite-list/src/terra-dev-site/test/infinite-list/InfiniteListUpdating.test.scss

```diff
@@ -0,0 +1,16 @@
+@import './InfiniteListTestCommon';
+
+:local {
```

Comment:
Could we just reuse the styles defined in `InfiniteListSameCount.test.scss` instead of adding another file to maintain?

Reply:
Created a single file "InfiniteListTestCommon.scss" and used this to replace all the individual sass files.